python/src/deltachat/account.py

```diff
@@ -104,3 +213,15 @@ def smtp_thread_run(self):
         while not self._thread_quitflag:
             capi.lib.dc_perform_smtp_jobs(self.dc_context)
             capi.lib.dc_perform_smtp_idle(self.dc_context)
+
+
+def convert_bytes(obj):
```

Comment:
convert what to what?  I mean naming and signature matters, docstrings help.  Same for ffi_unicode.  But also these two are clearly cffi implementation bits, the user shouldn't see this, why are these functions public?